src/modules/renderPage.js

```diff
@@ -3,7 +3,7 @@ import { Link } from '../helpers.js';
 const renderPage = function(page, popstate) {
 	document.documentElement.classList.remove('is-leaving');
 
-	const isCurrentPage = this.getCurrentUrl() === page.url;
```

Comment:
I feel like this shouldn't be part of this PR. Resolving cache entries and resolving the current url is two different things, no?

Reply:
This actually needs to be part of this PR, otherwise the new feature would not work.

Reply:
Mmh. Maybe I didn't understand the feature exactly? I thought this is about being able to cache similar results. I wouldn't expect a cache url resolver to resolve urls outside of the cache. If I have `/projects` and `/projects?filter=...` and both resolve to `/projects` in the cache, I would expect the cache to return the same html for either one; however I wouldn't expect it to stop rendering the new page entirely.